webapp/makefile

```diff
@@ -51,6 +51,8 @@ build-docker: ./Dockerfile
 		--build-arg react_app_jungle_testnet="$(REACT_APP_JUNGLE_TESTNET)" \
 		--build-arg react_app_lacchain_testnet="$(REACT_APP_LACCHAIN_TESTNET)" \
 		--build-arg react_app_disabled_menu_items='$(REACT_APP_DISABLED_MENU_ITEMS)' \
+		--build-arg react_app_network_url='$(REACT_APP_NETWORK_URL)' \
```

Comment:
@tetogomez for new env variables you need to pass it as a build argument (only for webapp)